sdk/python/feast/type_map.py

```diff
@@ -56,6 +95,9 @@ def python_type_to_feast_value_type(
     """
 
     type_name = type(value).__name__
+    if isinstance(value, list):
+        type_name = "ndarray"
+        value = np.asarray(value)
```

Comment:
This piece of code seems out of place. Can move closer/merge with the block starting at line 125?